pkg/quota/v1/evaluator/core/persistent_volume_claims_test.go

```diff
@@ -152,10 +156,23 @@ func TestPersistentVolumeClaimEvaluatorUsage(t *testing.T) {
 			},
 			enableRecoverFromExpansion: true,
 		},
+		"pvc-usage-by-volume-attributes-class": {
```

Comment:
Is it possible to have a test case of quota exceed?

Reply:
@sunnylovestiramisu added in https://github.com/kubernetes/kubernetes/pull/121895/commits/3f8590e918e2bc88dbda20ad2da8403644325792